### PR TITLE
[WIP] Figuring out nested controller with user_roles

### PR DESCRIPTION
**Pull request by allindow**

### CODE REVIEW

**Comment by allindow** [comment]
@jmejia

File: spec/models/role_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Role, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (jmejia) [inline spec/models/role_spec.rb]: @allindow Let's remove the blank specs from this PR or add some tests :)

File: spec/models/user_role_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe UserRole, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (jmejia) [inline spec/models/user_role_spec.rb]: @allindow See comment above.

**Comment by jmejia** [comment]
@allindow Looks like a solid PR. Clean code ✨

**Pull request closed without merging**